### PR TITLE
react-marketingsite

### PR DESCRIPTION
**Pull request by Victory-ET**
Code submission for Building a marketing site with form functionality with React and Webiny

### CODE REVIEW

**Review by endymion1818 — CHANGES_REQUESTED** [review]
Hi @Victory-ET , please rename the folder to the format `{tech}-{usecase}`, so for this it would be `react-marketingsite`

**Comment by Victory-ET** [comment]
Hi @endymion1818 I've made the required changes.

File: tutorials/react-marketingsite/src/index.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+html{
+    scroll-behavior: smooth;
+}
+body {
+  margin: 0;
+  /* font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif; */
```
Comment (endymion1818) [inline tutorials/react-marketingsite/src/index.css:13]: Please remove commented code or combine this with line 14 to provide font fallbacks.

**Review by endymion1818 — CHANGES_REQUESTED** [review]

File: tutorials/react-marketingsite/src/index.css (hunk 1 above)
Comment (Victory-ET) [reply tutorials/react-marketingsite/src/index.css]: Done @endymion1818

**Review by endymion1818 — APPROVED** [review]

**Pull request merged**